13-transport.Rmd

```diff
@@ -294,7 +291,7 @@ bristol_od$Active = (bristol_od$bicycle + bristol_od$foot) /
   bristol_od$all * 100
 ```
 
-There are two main types of OD pair: *interzonal* and *intrazonal*.
+There are two main types of OD pairs: *interzonal* and *intrazonal*.
```

Comment:
Initially I thought this change was wrong, but now think you're right: there are OD pairs come in many types. https://english.stackexchange.com/questions/29843/types-of-followed-by-singular-or-plural